server/routers/usersRouter.js

```diff
@@ -31,6 +44,37 @@ const users = require('../data/helpers/usersModel')
 =======
 }); 
 
+signin = (req, res) => {
+    //implementing user signin 
+    const creds = req.body;
+    users.get(creds.usersname)
```

Comment:
Should be `creds.email`.

Reply:
Singing will only use username at this time going forward this code can change we haven't fleshed out anything for signing and should focus on displaying users for Monday that's the MVP.

Reply:
Agreed. `username` is also spelled wrong.